src/mixins/object_interactivity.mixin.ts

```diff
@@ -172,11 +184,13 @@ export class InteractiveFabricObject extends FabricObject {
       dim = this._calculateCurrentDimensions(transformOptions),
       coords: Record<string, TOCoord> = {};
 
-    this.forEachControl(
-      (control: any, key: string, fabricObject: InteractiveFabricObject) => {
-        coords[key] = control.positionHandler(dim, finalMatrix, fabricObject);
```

Comment:
This is the only place `positionHandler`.
Meaning the signature was actually with 3 args and not 4

Reply:
9f3d5ea06